src/main.ts

```diff
@@ -26,7 +26,17 @@ export {
  */
 export { RobotClient as Client } from './robot';
 
-export { ViamClient } from './app/viam-client';
+export {
+  createViamClient,
+  type ViamClient,
```

Comment:
This is a breaking change - users won't be able to instantiate a `ViamClient` with the class constructor anymore and must use `createViamClient` instead.

This is more idiomatic and will make it easier to make non-breaking changes to interface moving forward.

We should do this for other class constructors as well but that's a thing for another day.